tests/test_raster_import_pixellimit.py

```diff
@@ -0,0 +1,159 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+#######
+
+"""
+Tests: Import raster with pixellimit check
+"""
+import unittest
+from flask.json import dumps as json_dumps
+
+try:
+    from .test_resource_base import (
+        ActiniaResourceTestCaseBase,
+        URL_PREFIX,
+        additional_external_data,
+    )
+except Exception:
+    from test_resource_base import (
+        ActiniaResourceTestCaseBase,
+        URL_PREFIX,
+        additional_external_data,
+    )
+
+__license__ = "GPLv3"
+__author__ = "Lina Krisztian"
+__copyright__ = "Copyright 2023, mundialis GmbH & Co. KG"
+__maintainer__ = "mundialis GmbH & Co. KG"
+
+
+class ImportRasterLayerPixellimitTestCase(ActiniaResourceTestCaseBase):
+    location = "nc_spm_08"
+    tmp_mapset = "mapset_rasterimport_pixellimit"
+    endpoint = f"/locations/{location}/mapsets/{tmp_mapset}/processing_async"
+    rimport_inp = "elevation"
+    # import resolution with which the process should fail:
+    rimport_res_fail = 0.1
+
+    def setUp(self):
+        # create new temp mapset
+        super(ImportRasterLayerPixellimitTestCase, self).setUp()
+        self.create_new_mapset(self.tmp_mapset, location_name=self.location)
+
+    def tearDown(self):
+        # delete mapset
+        self.delete_mapset(self.tmp_mapset, location_name=self.location)
+        super(ImportRasterLayerPixellimitTestCase, self).tearDown()
+
+    def test_pixellimit_not_reached(self):
+        """
+        Test import of raster, for which pixellimit is not reached
+        """
+        raster_url = additional_external_data[self.rimport_inp]
+        raster = self.rimport_inp
+        process_chain = {
+            "version": 1,
+            "list": [
+                {
+                    "id": "1",
+                    "module": "r.import",
+                    "inputs": [
+                        {
+                            "param": "input",
+                            "value": raster_url,
+                        },
+                        {
+                            "param": "output",
+                            "value": raster,
+                        },
+                    ],
+                },
+            ],
+        }
+        rv = self.server.post(
+            URL_PREFIX + self.endpoint,
+            headers=self.admin_auth_header,
+            data=json_dumps(process_chain),
+            content_type="application/json",
+        )
+        # Import should succeed
+        self.waitAsyncStatusAssertHTTP(
+            rv,
+            headers=self.admin_auth_header,
+            http_status=200,
+            status="finished",
+        )
+
+    def test_pixellimit_reached(self):
+        """
+        Test import of raster, for which pixellimit is reached
```

Comment:
Suggestion to add the negation here (again my first thought was the opposite of what happens):

```suggestion
    def test_pixellimit_not_allowed(self):
        """
        Test import of raster, for which pixellimit is reached and therefore not allowed
```